include/boost/beast/websocket/impl/stream_impl.hpp

```diff
@@ -128,8 +128,7 @@ struct stream<NextLayer, deflateSupported>::impl_type
             boost::empty_init_t{},
             std::forward<Args>(args)...)
         , detail::service::impl_type(
-            net::query(this->boost::empty_value<NextLayer>::get().get_executor(),
-                net::execution::context))
+            get_context(this->boost::empty_value<NextLayer>::get().get_executor()))
```

Comment:
N.B. In my local changes, I qualified this call with `this->` to ensure ADL wouldn't be used.